standard-covoiturage_openapi.yaml

```diff
@@ -379,6 +442,80 @@ components:
               type: integer
               description: If a passenger journey, requested seats.
 
+    CarpoolBookingPatch:
+      properties:
+        passengerPickupDate:
+          type: number
+          description: Passenger pickup datetime as a UNIX UTC timestamp in seconds.
+          format: long
+        passengerPickupLat:
+          type: "number"
+          format: "double"
+          description: Latitude of the passenger pick-up point.
+        passengerPickupLng:
+          type: "number"
+          format: "double"
+          description: Longitude of the passenger pick-up point.
+        passengerDropLat:
+          type: "number"
+          format: "double"
+          description: Latitude of the passenger drop-off point.
+        passengerDropLng:
+          type: "number"
+          format: "double"
+          description: Longitude of the passenger drop-off point.
+        passengerPickupAddress:
+          type: string
+          description: String representing the pickup-up address.
+        passengerDropAddress:
+          type: string
+          description: String representing the drop-off address.
+        status:
+          type: string
+          enum:
+            - WAITING_CONFIRMATION
+            - CONFIRMED
+            - CANCELLED
+            - COMPLETED_PENDING_VALIDATION
+            - VALIDATED
+        duration:
+          type: integer
+          description: Carpooling duration in seconds.
+        distance:
+          type: "integer"
+          description: |
+            Carpooling distance in meters. When the booking is COMPLETED or VALIDATED, this is the actual distance travelled if available.
+    CarpoolBooking:
+      required:
+      - id
+      - passengerId
+      - driver
+      - passengerPickupDate
+      - passengerPickupLat
+      - passengerPickupLng
+      - passengerDropLat
+      - passengerDropLng
+      - status
+      - price
+      allOf:
+        - $ref: '#/components/schemas/CarpoolBookingPatch'
+      properties:
+        id:
+          type: "string"
+          description: Unique identifier of the booking.
+        passengerId:
```

Comment:
Seems to me that the best practise in terms of compliance and security would be that a booking is created or updated using the end-user credentials for authentication (ie: _access_token_ for OIDC). In this flow, the operator would act "in the name of" the passenger to publish its bookings to the MaaS platform. So `passengerId` would be determined by the MaaS given the credentials used for authentication. And it wouldn't be possible to set or change the `passengerId`.